src/js/popup.js

```diff
@@ -66,11 +67,15 @@ function choosePanel(numRemaining, panelId, premium){
 }
 
 function checkUserSubdomain(premiumSubdomainSet){
-  if(premiumSubdomainSet != "None"){
-    document.getElementsByClassName("register-domain-component")[0].remove();
+  const educationalComponent = document.querySelector(".educational-component");
+  const registerDomainComponent = document.querySelector(".register-domain-component");
+
+  if (premiumSubdomainSet != "None") {
+    registerDomainComponent.classList.add("is-hidden");
```

Comment:
Since all the premium UI blocks have `is-hidden` removed from them when they're premium, depending on whether or not a user has a subdomain, this line will determine which module to display by hiding the opposing one (register domain CTA vs educational module).